tests/test_pipeline.py

```diff
@@ -67,18 +67,18 @@ def test_ge_le_gt_lt(
 @pytest.mark.parametrize(
     'type_, pipeline, valid_cases, invalid_cases',
     [
-        (int, validate_as(int).multiple_of(5), [5, 20, 0], [18, 7]),
-        (float, validate_as(float).multiple_of(2.5), [2.5, 5.0, 7.5], [3.0, 1.1]),
+        (int, validate_as(int).multiple_of(5), [5, 20, 0], [18, 7]),  # type: ignore
```

Comment:
Why do we use an `ignore` here?

Reply:
Hmm I was getting type errors but can't reproduce anymore. Will remove.